packages/lwt/lwt.2.5.1/opam

```diff
@@ -40,5 +40,6 @@ depopts: [
 conflicts: [
  "react" {<"1.0.0"}
  "ssl" {< "0.5.0"}
+ "camlp4" {= "4.02+7"} (* lacks "camlp4.quotations.o" *)
```

Comment:
Same here, Everything installed and it works fine. [camlp4 still has quotation.o](https://github.com/ocaml/camlp4/blob/4.02/camlp4/META.in#L66-L77).
I'm pretty sure this conflict is wrong. 

Next time, could you at the very least ping lwt people, or report the bug upstream when you change the opam packages ?